libgdf/python/librmm_cffi/wrapper.py

```diff
@@ -85,6 +86,52 @@ def finalize(self):
         """
         return self.rmmFinalize()
 
+    def _get_caller(self):
+        """Finds the file and line number of the caller (first caller outside
+           this file.)
+        """
+        stack = inspect.stack()
+        # Go up stack to find first caller outside this file (more useful)
+        caller_i = next(x for x in range(len(stack)) 
+                        if 'wrapper.py' not in stack[x][1])
```

Comment:
I don't suspect we'd see any performance issues with this given the stack should be relatively small, but wanted to comment in case it does come up. Also could possibly use an `endswith` instead of in to make the string matching faster.